Makefile

```diff
@@ -70,12 +70,11 @@ USER_NAME ?= $(shell id -un)
 GIT_COMMIT_HASH ?= $(shell git rev-parse --verify HEAD)
 GIT_COMMIT_DATE ?= $(shell git log -1  --date=iso --pretty=format:%cd)
 CURRENT_DATE ?= $(shell date -u +"%Y-%m-%d %H:%M:%S %z")
-DEPLOY_GIT_BRANCH ?= master
 S3_VERSION_PATH ?=
 S3_MF_GEOADMIN3_INFRA = mf-geoadmin3-infra-dublin
 S3_BASE_PATH ?=
 S3_SRC_BASE_PATH ?=
-CLONEDIR = /home/$(USER_NAME)/tmp/branches/${DEPLOY_GIT_BRANCH}
+CLONEDIR = /home/$(USER_NAME)/tmp/branches/${GIT_BRANCH}
```

Comment:
GIT_BRANCH is taken from `.build-artefacts/last-deploy-git-branch` and therefore will be set to master most of the time.
My take would be to adapt, `clonebuilds.sh` directly instead or default S3DEPLOY_GIT_BRANCH to the current branch.